MobileCenter/MobileCenter/MSMobileCenter.m

```diff
@@ -154,7 +154,7 @@ - (BOOL)configure:(NSString *)appSecret {
   }
 
   // Validate and set the app secret.
-  else if ([appSecret length] == 0 || ![[NSUUID alloc] initWithUUIDString:appSecret]) {
+  else if ([appSecret length] == 0) {
```

Comment:
It's there already, if appSecret is nil then we end up with nil==0 which is true in objective c.